learn/quick-start.md

```diff
@@ -1,7 +1,143 @@
 # Quick start
 
- Work in progress :construction:
+ To start using Codex we would need to perform the following steps:
+ - [Review the disclaimer](/codex/disclaimer)
+ - [Get Codex binary](#get-codex-binary)
+ - [Run Codex](#run-codex)
+ - [Interact with Codex](#interact-with-codex)
 
- Please read
- - [Codex Two-Client Test](https://github.com/codex-storage/nim-codex/blob/master/docs/TwoClientTest.md)
- - [Running a Local Codex Network with Marketplace Support](https://github.com/codex-storage/nim-codex/blob/master/docs/Marketplace.md)
+### Get Codex binary
+
+ For quick start we will use precompiled binaries from [GitHub release page](https://github.com/codex-storage/nim-codex/releases). If you prefer to compile from the sources, please check [Build Codex](/learn/build).
+
+ 1. Download binary and checksum for your platform/architecture
+
+    **Linux/macOS**
+    ```shell
+    version=v0.1.3
+    platform=linux     # darwin
+    architecture=amd64 # arm64
+
+    # Binary
+    curl -LO https://github.com/codex-storage/nim-codex/releases/download/${version}/codex-${version}-${platform}-${architecture}.tar.gz
+
+    # Checksum
+    curl -LO https://github.com/codex-storage/nim-codex/releases/download/${version}/codex-${version}-${platform}-${architecture}.tar.gz.sha256
+    ```
+
+ 2. Verify checksum
+    ```shell
+    # Linux
+    sha256sum -c codex-${version}-${platform}-${architecture}.tar.gz.sha256
+
+    # macOS
+    shasum -a 256 -c codex-${version}-${platform}-${architecture}.tar.gz.sha256
+    ```
+    Make sure you get `OK` in the result
+    ```
+    codex-v0.1.3-linux-amd64.tar.gz: OK
+    ```
+
+ 3. Extract binary
+    ```shell
+    tar -zxvf codex-${version}-${platform}-${architecture}.tar.gz
+    ```
+
+ 4. Copy binary to the appropriate folder
+    ```shell
+    sudo install codex-${version}-${platform}-${architecture} /usr/local/bin/codex
+    ```
+
+ 5. Install dependencies
+    ```shell
+    # Debian-based Linux
+    sudo apt update && sudo apt install libgomp1
+    ```
+
+ 6. Check the result
+    ```shell
+    codex --version
+    ```
+    ```shell
+    Codex version:  v0.1.3
+    Codex revision: 89917d4
+    Nim Compiler Version 1.6.14 [Linux: amd64]
+    ```
```

Comment:
my docker container gives me:
root@fe34335b0e8c:/usr/local/bin# ./codex --version
./codex: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./codex)
./codex: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./codex)
./codex: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./codex)
./codex: /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by ./codex)

Reply:
Is it ubuntu `20.04` or `22.04`?

On `24.04` all should work, but let me check.